### DISCUSSION TITLE
show/hide plugin from command line?

### DISCUSSION THREAD

**Discussion by lilruffian** [FelixKratz/SketchyBar#74] [Q&A] [answered]
Enjoying this rewrite @FelixKratz, nicely done!

I have a plugin which displays my current [doing](https://github.com/ttscoff/doing) task item. It works great, however, I'd like to hide the plugin from new when I'm not in focus mode. Is there a way to hide / show a plugin using the command line? 

Thanks to anyone who can help me out!

**Comment by FelixKratz** [comment, marked as answer]
Sure, you can simply disable *drawing* of the item:
```bash
sketchybar -m --set <name> drawing=off
```
and reenable it via:
```bash
sketchybar -m --set <name> drawing=on
```
It is also possible to toggle the drawing state via:
```bash
sketchybar -m --set <name> drawing=toggle
```
Hope this answers your question.
Edit: Update to v2.0.0 Syntax

**Comment by lilruffian** [comment]
That works great! Thanks!!

**Comment by BrunnoVert** [comment]
Is possible to toggle the bar with this code?
